package.json

```diff
@@ -70,13 +70,12 @@
     "css-loader": "^6.7.1",
     "eslint": "^7.3.1",
     "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-jsdoc": "^37.7.0",
```

Comment:
Had to be updated as 37 wouldn't work with newer node version.

As a result, had to update the eslint rules to a slighltly more strict one and is why you see file changes that is just removing some whitespace.